examples/grafana-alloy-auto-instrumentation/ebpf-otel/README.md

```diff
@@ -67,10 +39,10 @@ minikube image load test-ebpf-profiler:latest
 2. Deploy to Kubernetes:
 ```bash
```

Comment:
What errors did you find on `minikube`? Didn't it run at all?
It worked to me when I tested it some weeks ago... 😅

Reply:
it shows only kernel stacktraces. The minikube is running in a nested pid namespace and the ebpf program is reporting processess with PIDs from host pid namesapce and userspace is unable to find such processes